chapters/classes.tex

```diff
@@ -408,15 +408,15 @@ \subsection{Conditional Component Declaration}\label{conditional-component-decla
 and the condition attribute is kept from the original declaration (see
 \cref{interface-compatibility-or-subtyping}).
 
-If the \lstinline!Boolean! expression is false the component (including its modifier) is removed from the flattened DAE, and connections to/from the component are removed.
+If the \lstinline!Boolean! expression is \lstinline!false!, the component (including its modifier) is removed from the flattened DAE, and connections to/from the component are removed.
 A component declared with a condition-attribute can only be modified and/or used in connections.
 
 \begin{nonnormative}
 Adding the component and then removing it ensures that the component is valid.
 
 If a \lstinline!connect!-equation defines the connection of a non-conditional component \lstinline!c1! with a conditional component \lstinline!c2! and \lstinline!c2! is de-activated, then \lstinline!c1! must still be a declared element.
 
-There are annotations to handle the case where the connector should be connected when activated, see \cref{annotations-for-the-graphical-user-interface}.
+There are annotations to handle the case where the connector should be connected when activated, see \cref{connection-restrictions}.
```

Comment:
I could not find any annotations in graphical user interface that would apply to this case. I think it should be conneciton-restrictions.

Reply:
Yes, I must have missed that they were in that section.